qutebrowser/misc/throttle.py

```diff
@@ -101,3 +101,54 @@ def set_delay(self, delay_ms: int) -> None:
     def cancel(self) -> None:
         """Cancel any pending instance of this timer."""
         self._timer.stop()
+
+
+class Delayed(QObject):
+
+    """A throttle to throttle calls.
+
+    If a request comes in, it will be processed after a timeout ends.
+    If another request comes in, the timer will be reset and the previous
+    request will be canceled.
+    """
```

Comment:
Before I review anything else, can you elaborate on what this class is actually intended to do? This docstring is obviously wrong, but the code below confuses me as well...

Reply:
Why is that wrong?

Reply:
It's copied from `Throttle`, yet this apparently is intended to do something other than `Throttle` (I just don't understand *what* exactly).

Reply:
No, I modified it. Not just copied.

Reply:
It still says "A throttle to throttle calls" though.